tests/v3/test_group.py

```diff
@@ -681,152 +680,22 @@ async def test_asyncgroup_update_attributes(store: Store, zarr_format: ZarrForma
     assert agroup_new_attributes.attrs == attributes_new
 
 
-async def test_group_members_async(store: LocalStore | MemoryStore) -> None:
```

Comment:
@jhamman was this `test_group_members_async` test moved somewhere, or deliberately removed?

Reply:
Not deliberately removed. Must have been a bad merge conflict resolution. I'll bring it back today. Sorry!